External/FEXCore/Source/Interface/Core/X86Tables/X86Tables.h

```diff
@@ -45,26 +46,10 @@ static inline void GenerateTable(X86InstInfo *FinalTable, U8U8InfoStruct const *
   }
 };
 
-static inline void GenerateTable(X86InstInfo *FinalTable, U16U8InfoStruct const *LocalTable, size_t TableSize) {
+template<typename T, typename TT>
```

Comment:
The first template parameter will cause separate versions of this functions to be generated for each of the arrays it's called with (currently just three but it's an easily avoidable trap). Better to keep using the previous `X86InstInfo*` pointer instead, or `std::span` if that's available for our current targets?

Reply:
Kept the previous raw pointer instead.